devices/ePaper/ePaper.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>nanoFramework.Iot.Device.ePaper</id>
+    <version>$version$</version>
+    <title>nanoFramework.Iot.Device.ePaper</title>
+    <authors>nanoFramework</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="file">LICENSE.md</license>
+    <releaseNotes>
+    </releaseNotes>
+    <readme>docs\README.md</readme>
+    <developmentDependency>false</developmentDependency>
+    <projectUrl>https://github.com/nanoframework/nanoFramework.IoT.Device</projectUrl>
+    <icon>images\nf-logo.png</icon>
+    <repository type="git" url="https://github.com/nanoframework/nanoFramework.IoT.Device" commit="$commit$" />
+    <copyright>Copyright (c) .NET Foundation and Contributors</copyright>
+    <description>This package includes .Net nanoFramework drivers for EPD devices with a graphics library. This version supports only displays based on the SSD1681 controller. More controllers will be added in the future. Please refer to the README file for more info about support.</description>
+    <summary>Iot.Device.ePaper assembly for .NET nanoFramework C# projects</summary>
+    <tags>nanoFramework C# csharp netmf netnf epd ePaper Iot.Device.Display.ePaper</tags>
+    <dependencies>
+      <dependency id="nanoFramework.CoreLibrary" version="1.12.0" />
+      <dependency id="nanoFramework.System.Device.Gpio" version="1.1.22" />
+      <dependency id="nanoFramework.System.Device.Spi" version="1.3.22" />
+      <dependency id="nanoFramework.System.Drawing" version="1.2.141" />
```

Comment:
Missing nanoFramework.Runtime.Events dependency which is required according to nfproj file :)